test/test_p4est_3d.jl

```diff
@@ -257,6 +257,35 @@ end
     end
 end
 
+@trixi_testset "elixir_euler_free_stream_boundaries_float32.jl" begin
+    # Expected errors are taken from elixir_euler_free_stream_boundaries.jl
+    @test_trixi_include(joinpath(EXAMPLES_DIR,
+                                 "elixir_euler_free_stream_boundaries_float32.jl"),
+                        l2=[
+                            Float32(6.530157034651212e-16),
+                            Float32(1.6057829680004379e-15),
+                            Float32(3.31107455378537e-15),
+                            Float32(3.908829498281281e-15),
+                            Float32(5.048390610424672e-15),
```

Comment:
Are these double precision values (and those in other tests) copied from the `Float64` version of the test?

Reply:
Yes, exactly - a few lines above, I have added the comment
```julia
# Expected errors are taken from elixir_euler_free_stream_boundaries.jl
```